### PR TITLE
Makes startup significantly faster

### PR DESCRIPTION
**Pull request by marlyn-x86**
Does so by the following methods:
-	Removes a `spawn` to call `Entered` for every atom
-	Makes all roundstart areas viable for `teleportlocs`
-	Removes a |= into a large list in `Turf`'s new

### CODE REVIEW

**Comment by bloxgate** [comment]
One issue we'll have to fix later, but we really need this so merging. @caelaislinn please make an issue for the issue.

File: code/game/turfs/turf.dm [hunk 1]
```diff
@@ -29,11 +29,12 @@
...
-	for(var/atom/movable/AM as mob|obj in src)
-		spawn( 0 )
-			src.Entered(AM)
-			return
-	turfs |= src
+	// for(var/atom/movable/AM as mob|obj in src)
+	// 	// spawn( 0 )
+	// 	// 	src.Entered(AM)
+	// 	// 	return
+	// Since `New` is only called once, we can safely use
```
Comment (caelaislinn) [inline code/game/turfs/turf.dm:36]: Turning turfs into open space needs this so that they can fall straightaway

Comment (marlyn-x86) [reply code/game/turfs/turf.dm:36]: Wouldn't that belong in a `changeTurf`, without the `spawn`?

Comment (marlyn-x86) [reply code/game/turfs/turf.dm:36]: https://github.com/HaloSpaceStation/HaloSpaceStation13/blob/alpha/code/game/turfs/turf_changing.dm#L42-L47
Looks like you're already covered

**Pull request merged**